widlparser/constructs.py

```diff
@@ -1914,7 +1914,7 @@ def __init__(self, tokens: Tokenizer, parent: ChildProduction) -> None:
 		Construct.__init__(self, tokens, parent, False)
 		self._attribute = Identifier(tokens)
 		self._open_paren = Symbol(tokens, '(')
-		self._arguments = ArgumentList(tokens, self) if (ArgumentList.peek(tokens)) else None
+		self._arguments = ArgumentList(tokens, self)
```

Comment:
Have you tested this thoroughly? I'm not sure and AgurmentList will construct properly if the peek fails (e.g. It may have a Single Argument...)

Reply:
I moved the peek check to the constructor and do an early exit if it fails, so you just get an ArgumentList with an empty list of args and commas.

(Having a single argument shouldn't change anything here; if it's wrong for that case now, it was wrong for that case before this PR, too.)